tests/test_rio_merge.py

```diff
@@ -556,7 +558,19 @@ def test_merge_precision(tmpdir, precision):
     runner = CliRunner()
     result = runner.invoke(main_group, ["merge", "-f", "AAIGrid"] + precision + inputs + [outputname])
     assert result.exit_code == 0
-    assert open(outputname).read() == textwrap.dedent(expected)
+
+    # The arrangement of whitespace in the data part of the file
```

Comment:
why does RasterIO look at the "internals" of GDAL managed formats ;-) ?  Can't it just check the pixel values comparing numpy arrays... ?

Reply:
Yes, I wouldn't dare do this with any other format. I do usually open the outputs with `rasterio.open()` and assert on attributes and raster data values. I thought the AAIGrid was perfectly stable and might be a good "second opinion" 😄

Reply:
I think that both methods are useful depending on the goals of the test. I don't see anything driver specific required for this test and I hope that no one is using AAIGrid in production 😀, so I think switching to numpy arrays would be perfectly fine here. If nothing else, it would make maintaining the test a little easier.

However, it has been useful in finding a change in GDAL to be verified that a bug wasn't introduced. So, I think that it would be fine to leave it in if you think that your time could be better used elsewhere.

Reply:
Since the `precision` parameter is already deprecated, how about we leave this test as it is, and delete it at 2.0?

Reply:
Sounds like a good plan 👍